plugins/wazo-gigaset/N870-83.v2.17.2/common.py

```diff
@@ -182,5 +182,20 @@ def is_sensitive_filename(self, filename):
 
     _SENSITIVE_FILENAME_REGEX = re.compile(r'^[0-9a-f]{12}\.xml$')
 
-    def synchronize(self, device, raw_config):
-        return synchronize.standard_sip_synchronize(device)
+    if hasattr(synchronize, 'standard_sip_synchronize'):
+        def synchronize(self, device, raw_config):
+            return synchronize.standard_sip_synchronize(device)
+
+    else:
+        # backward compatibility with older xivo-provd server
```

Comment:
How old?

Reply:
Not sure, it was a copy-paste from other plugins... Maybe we could remove it